doc/nrf/config_and_build/dfu/index.rst

```diff
@@ -10,20 +10,39 @@ Device Firmware Updates
 
 Device Firmware Update (DFU) is the procedure of upgrading the application firmware version on a device.
 It is composed of two steps.
-First a new firmware image is transferred to the chip, and then the :ref:`bootloader <app_bootloaders>` tests and boots the new firmware.
+First, a new firmware image is transferred to the chip, and then the :ref:`bootloader <app_bootloaders>` tests and boots the new firmware.
 
-You can transfer the updated images to the device in two ways:
+.. note::
+  The choice of bootloader affects how firmware updates can be performed.
+  While specifics regarding bootloader features such as out-of-the-box support for various DFU methods may vary, it's generally recognized that:
 
-* Wired, where updates are sent through a wired connection, like UART, or delivered by connecting a flash device.
-* Over-the-air (OTA), where updates are sent through a wireless connection, like Bluetooth® Low Energy.
+  * MCUboot is designed with flexibility in mind and supports various protocols and methods for firmware updates.
+    However, the extent of supported features, like USB-DFU class, may vary and should be confirmed with the latest MCUboot documentation or source code.
+  * The nRF Secure Immutable Bootloader (NSIB) ensures secure firmware updates but might not include comprehensive support for all types of firmware updates compared to MCUboot.
```

Comment:
Nitpick: 
The reader might think: "Why do they specify that NSIB ensures secure DFU? That looks like MCUboot is not as secure?"

Reply:
Changed line 21 to the following:

`* The nRF Secure Immutable Bootloader (NSIB) allows for firmware updates but might not include comprehensive support for all types of firmware updates compared to MCUboot.
    Specifics of NSIB’s capabilities, particularly in terms of direct DFU support, are subject to change and should be verified with the latest documentation.
`